dojo/finding/views.py

```diff
@@ -2446,17 +2446,10 @@ class Original(ImageSpec):
 
     try:
         access_token = FileAccessToken.objects.get(token=token)
-        size = access_token.size
-
         if access_token.size not in list(size_map.keys()):
             raise Http404
         size = access_token.size
```

Comment:
I guess this line can be removed too @Maffooch , right?

Reply:
Yeah it looks like it is not used anymore either

Reply:
ah, it is used in line 2458:
image = size_map[size](source=file).generate()